src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSessionDetailsPanel.tag

```diff
@@ -30,7 +30,7 @@
                 <div class="form-group">
                     <label class="col-sm-2 control-label">Closing time:</label>
                     <div class="col-sm-10">
-                        <p class="form-control-static">${feedbackSession.endTimeString}</p>
+                        <p class="form-control-static" id="end-time">${feedbackSession.endTimeString}</p>
```

Comment:
Ahh okay, I see. Any suggestions on converting to UTC in time helper? The `endTime` `java.util.Date` object from `FeedbackSessionAttributes` contains no timezone information at all so this will be slighty tricky.